.github/workflows/ci.yml

```diff
@@ -37,12 +39,14 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-        with: 
+        with:
           repository: waku-org/js-waku
       - uses: actions/setup-node@v3
         with:
           node-version: ${{ env.NODE_JS }}
       - uses: ./.github/actions/npm
+      - name: Install headless-tests dependencies
```

Comment:
isn't `Install headless-tests dependencies` needed only for `browser-test` step?